### PR TITLE
Stretch dovecot: dovecot.conf et Postfix :master.cf

### PR DESCRIPTION
**Pull request by kitoy30**
Dovecot:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=844271

Postfix : 
Ajout de la connexion smtp over tls
Ajout de la connexion submission (STARTTLS)

### CODE REVIEW

**Comment by taziden** [comment]
Dovecot commit looks good to me.
About Postfix, I'd like to review the whole master.cf conf and I'll definitely take your commit into acount.

**Comment by alexAubin** [comment]
@taziden : did you manage to find time to look at postfix conf ? :sweat_smile: 

In the meantime, I think I'm gonna cherry-pick/merge https://github.com/YunoHost/yunohost/pull/397/commits/1b191db711975e5d32b293e399f7be7a2dfdb0c6 since there seem to be a consensus on this ( and  @Josue-T just encountered the issue)

File: data/templates/postfix/plain/master.cf [hunk 1]
```diff
@@ -10,6 +10,16 @@
 #               (yes)   (yes)   (no)    (never) (100)
 # ==========================================================================
 smtp      inet  n       -       y       -       -       smtpd
+
```
Comment (taziden) [inline data/templates/postfix/plain/master.cf]: smtps on port 465 is deprecated for years. We shouldn't support it anymore.

**Comment by alexAubin** [comment]
Thanks for the feedback ! :heart: 

Not an mail expert, but that seem to make sense.

@kitoy30 , are you okay with that ?

**Comment by alexAubin** [comment]
So we get this for master.cf : https://github.com/YunoHost/yunohost/pull/397/files#diff-889aa53e4712257d7a64d4aee860b86d which should implement the comments from @taziden 

@kitoy30 & @taziden : are you okay with this ?

**Comment by taziden** [comment]
OK with this !

File: data/templates/nginx/plain/yunohost_admin.conf [hunk 2]
```diff
@@ -25,8 +25,9 @@ server {
     # Ciphers with intermediate compatibility
     # https://mozilla.github.io/server-side-tls/ssl-config-generator/?server=nginx-1.6.2&openssl=1.0.1t&hsts=yes&profile=intermediate
     ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-    ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS';
+    ssl_ciphers 'kEECDH+ECDSA+AES128 kEECDH+ECDSA+AES256 kEECDH+AES128 kEECDH+AES256 +SHA !aNULL !eNULL !LOW !MD5 !EXP !DSS !PSK !SRP !kECDH !CAMELLIA !RC4 !SEED';
```
Comment (frju365) [inline data/templates/nginx/plain/yunohost_admin.conf]: Duplicate to another PR. DIscussion needed.

Comment (alexAubin) [reply data/templates/nginx/plain/yunohost_admin.conf]: Yes, not sure what happened with this PR / branch exactly, but I think I'll just cherry-pick the relevant commits

**Comment by alexAubin** [comment]
Okay, I cherry-picked 2064dcc21b63068a8d396a27a0daa819d704f2bd and b3cece69fbfe82c4c409c419956c8673452e2fd1 into the stretch branch ! (c.f. https://github.com/YunoHost/yunohost/commits/38adc8905974766209d0321de47ae8c67944098a/data/templates/postfix/plain/master.cf)

Closing this PR (sorry for not being able to do a regular merge but there's too many weird conflicts due to many rebases of the stretch branch)

Thanks for the contribution and discussion ! :heart:

**Pull request closed without merging**